### PR TITLE
P rupdates

### PR DESCRIPTION
**Pull request by e-t-trudell**
Resolved pull request conflicts from PR#6
https://github.com/JEub/dev-this-address-book/pull/6

Updated read me in both root and frontend directories
Added Nav.css to condense styling for easier imports instead of using react styled components
removed extra white space in App.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,17 @@
   - Address
 - Home page
 - Update addresses
-- Colors: {Black. Lime Green. Light Grey}
\ No newline at end of file
+- Colors: {Black. Lime Green. Light Grey}
+
+To run frontend from the terminal: cd frontend cd addressbook npm run dev
```
Comment (JEub) [inline README.md]: Start this section as a new section in the readme. Call it 

## Getting Started

To run the front end application in a unix terminal

```bash
# change into the address book directory
cd ./frontend/addressbook/

# run the development environment for hot reloads
npm run dev
```

File: frontend/address-book/src/App.jsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 import LandingPage from "./views/LandingPage";
 
 function App() {
-  return (
-    <>
-      <h1>Pawnee Parks and Recs Dept.</h1>
-      <LandingPage />
-    </>
-  );
+    return (
```
Comment (JEub) [inline frontend/address-book/src/App.jsx:6]: Make sure to pull and merge from main.

File: frontend/address-book/src/components/Nav.jsx [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
+                <h3 className="unit">Pawnee: Parks and Rec Dept.</h3>
+                {/* paths needed for to */}
+                {/* <Link to="/api/newCustomer">
+                Add Customer
+            </Link>
+            <Link to="/api/logout">
+                Logout
+            </Link> */}
+                <button id="button">
+                    <a href="/">Add Customer</a>
```
Comment (JEub) [inline frontend/address-book/src/components/Nav.jsx]: According to @parradaniela "a button should do something, and an anchor should take you somewhere. You should avoid using both in the same area."

File: frontend/address-book/src/components/SearchBar.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    const [searchInput, setSearchInput]= useRecoilState(searchInputState);
+    const [currentSearchValue,setCurrentSearchValue]= useRecoilState(currentSearchTerm );
+    // connect to database for search results?
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        console.log(searchInput);
+        setCurrentSearchValue(searchInput);
+        setSearchInput("");
+    };
+    // const button = {
```
Comment (JEub) [inline frontend/address-book/src/components/SearchBar.jsx]: Lets get rid of these cSs comments

File: frontend/address-book/src/utils/API.jsx
Comment (JEub) [inline frontend/address-book/src/utils/API.jsx:1]: Remove!

**Review by JEub — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (e-t-trudell) [reply README.md]: Corrected and pushed

File: frontend/address-book/src/App.jsx (hunk 2 above)
Comment (e-t-trudell) [reply frontend/address-book/src/App.jsx:6]: Corrected and pushed

File: frontend/address-book/src/components/Nav.jsx (hunk 3 above)
Comment (e-t-trudell) [reply frontend/address-book/src/components/Nav.jsx]: Corrected and pushed

File: frontend/address-book/src/components/SearchBar.jsx (hunk 4 above)
Comment (e-t-trudell) [reply frontend/address-book/src/components/SearchBar.jsx]: Corrected and pushed

File: frontend/address-book/src/utils/API.jsx
Comment (e-t-trudell) [reply frontend/address-book/src/utils/API.jsx:1]: Corrected and pushed

**Comment by e-t-trudell** [comment]
Updated changes requested from yesterdays code review. Changes include updating treadles, removing npm package for axis that was not needed, removing css comments, pulling main branch to have updated frontend changes.

**Review by JEub — APPROVED** [review]

**Review by JEub — APPROVED** [review]

**Pull request merged**